src/main/java/org/jenkinsci/plugins/JiraTestResultReporter/JiraUtils.java

```diff
@@ -230,35 +236,30 @@ public static SearchResult findIssues(Job project, TestResult test, EnvVars envV
     /**
      * Escape the JQL query of special characters.
      *
-     * Currently:
-     *  + - & | ! ( ) { } [ ] ^ ~ * ? \ / :
-     *
-     * Reference:
-     *  https://confluence.atlassian.com/jiracoreserver073/search-syntax-for-text-fields-861257223.html
+     * Based on https://jira.atlassian.com/browse/JRASERVER-25092, currently supported:
+     *  + - & | ~ *
+     *  
+     * Unsupported:
+     *  ! ( ) { } ^ ? \ /
      *
+     * Provides special support for parameterized tests [ ] 
```

Comment:
To be clear, there is no magic.
```suggestion
     * Provides special support for parameterized tests by ignoring the parameter in [ ] 
```